test/jdk/com/sun/jdi/DataDumpTest.java

```diff
@@ -93,7 +98,7 @@ private static void runTest(String jdwpArg) throws Exception {
             out.waitFor(); // Wait for the debuggee to exit
 
             System.out.println("Deuggee output:");
```

Comment:
nit typo: s/Deuggee/Debuggee/

Not your typo, but could you fix it while you are there?